README.md

```diff
@@ -1,19 +1,127 @@
+## Background / Motivation
 
-There are many requests for adding number-formatting-related features to ECMA 402.  A few of them include:
+There are many requests for adding number-formatting-related features to ECMA 402. These include:
 
-- tc39/ecma402#200
-- tc39/ecma402#186
-- tc39/ecma402#164
-- tc39/ecma402#163
-- tc39/ecma402#95
-- tc39/ecma402#91
-- tc39/ecma402#37
-- tc39/ecma402#32
+- [Expose narrow currency symbol](https://github.com/tc39/ecma402/issues/200) -- Section I
+- [Add currency accounting format](https://github.com/tc39/ecma402/issues/186) -- Section IV
+- [Add scientific notation](https://github.com/tc39/ecma402/issues/164) -- Section III
+- [Add option to force sign](https://github.com/tc39/ecma402/issues/163) -- Section IV
+- [Add compact decimal notation](https://github.com/tc39/ecma402/issues/37) -- Section III
+- [Add measure unit formatting](https://github.com/tc39/ecma402/issues/32) -- Section II
 
-Rather than complicate `Intl` with more subclasses with heavilly overlapping functionality, this proposal is to restructure the spec of `Intl.NumberFormat` to make it more easilly support additional features in a "unified" way.
+These features are important to both end users and to Google.  Since most of these features require carrying along large amounts of locale data for proper i18n support, exposing these features via a JavaScript API reduces bandwidth and lowers the barrier to entry for i18n best practices.
+
+Rather than complicate `Intl` with more constructors with heavilly overlapping functionality, this proposal is to restructure the spec of `Intl.NumberFormat` to make it more easilly support additional features in a "unified" way.
+
+Additional background: [Unified API for number formatting](https://github.com/tc39/ecma402/issues/215)
+
+## I. Spec Cleanup
+
+Certain sections of the spec have been refactored with the following objectives:
+
+- Fix https://github.com/tc39/ecma402/issues/238 (currency long name has dependency on plural form, and the currency long name pattern has dependency on currencyWidth).
+- Move pattern resolution out of the constructor to keep all internal fields of NumberFormat locale-agnostic, making it easier to reason about behavior in the format method.
+
+In addition, one missing option is added to the existing `currencyDisplay` setting: "narrowSymbol", which uses the CLDR narrow-format symbol.
+
+## II. Measure Units
+
+Units of measurement can be formatted as follows:
+
+```javascript
+(9.81).toLocaleString("en-US", {
+    style: "unit",
+    unit: "acceleration-meter-per-second-squared",
+    unitDisplay: "short"
+});
+// ==> "9.81 m/s²"
+```
+
+The syntax was discussed in #3.
+
+- `style` receives the string value "unit"
+- `unit` receives a string measure unit identifier, defined in [UTS #35](http://unicode.org/reports/tr35/tr35-general.html#Unit_Elements).  See also the [full list of unit identifiers](https://unicode.org/repos/cldr/tags/latest/common/validity/unit.xml).
```

Comment:
@waldemarhorwat has previously objected to a previous unit list as it wasn't arbitrary generative. It's good that you have the cross-reference to CLDR here; this rationale (that this is the shape of the data we have) may be good to highlight in the presentation.

Reply:
I mean, not allowing unrestricted numerator and denominator units which are each products of other units.

Reply:
Thanks for the clarification.  I reworded my discussion.  Since proper support for compound units requires more design work and popular compound units like meters per second are already built in to CLDR directly, I do not plan to add them to this proposal.

Reply:
That sounds good to me. If you can include this explanation in your presentation, it will be helpful.